packaging/rpm/microshift.spec

```diff
@@ -137,6 +137,17 @@ Requires: microshift = %{version}
 %description olm
 The microshift-olm package provides the required manifests for the Operator Lifecycle Manager to be installed on MicroShift.
 
+%package olm-release-info
+Summary: Release information for Operator Lifecycle Manager components for MicroShift
+BuildArch: noarch
+Requires: microshift-release-info = %{version}
```

Comment:
This does not correspond to the description of the PR. I guess, we need to fix the description because the code seems to be OK.

Reply:
fixed